cmd/earthly/main.go

```diff
@@ -1176,6 +1176,86 @@ Set up a whole custom git repository for a server called example.com, using a si
 				},
 			},
 		},
+		{
+			Name: "preview",
+			Subcommands: []*cli.Command{
+				{
+					Name:        "project",
+					Description: "Create and manage Earthly projects",
+					UsageText:   "earthly project (ls|rm|create|member)",
+					Flags: []cli.Flag{
+						&cli.StringFlag{
+							Name:     "org",
+							EnvVars:  []string{"EARTHLY_ORG"},
+							Usage:    "The name of the organization to which the project belongs. Required when user is a member of multiple.",
+							Required: false,
+						},
+					},
+					Subcommands: []*cli.Command{
+						{
+							Name:        "ls",
+							Aliases:     []string{"list"},
+							Usage:       "List all projects that belong to the specified organization",
+							Description: "List all projects that belong to the specified organization",
+							UsageText:   "earthly project --org <organization-name> ls",
```

Comment:
should we give a hint that org is optional for most users? e.g.
```suggestion
							UsageText:   "earthly project [--org <organization-name>] ls",
```

Reply:
Can do, although I didn't realize the org would be optional at this point. I figured we would be able to set it with the `PROJECT` command but not until that's ready. Is there a way to set the project without this flag in the current CLI version?

Reply:
We could follow the convention we have on the `satellite` commands - org only needs to be specified if you're a member of more than one. Also we have been using an `EARTHLY_ORG` env for those commands as an alternative to specifying `--org`.

Another thought, we I don't think the UsageText has the square brackets like that to show it's optional for `satellite` commands but it could be nice to update it there as well

Reply:
That seems like a decent heuristic. I can add support for that now and see where things settle with the `PROJECT` command (and other options).